@here/olp-sdk-core/lib/utils/MultiPartUploadWrapper.ts

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (C) 2021 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { BlobData, OlpClientSettings, TypedEvent } from "@here/olp-sdk-core";
+
+/**
+ * Contains information about the uploaded chunk.
+ */
+export interface UploadStatus {
+    /**
+     * The total count of the uploaded chunks.
+     */
+    uploadedChunks: number;
+    /**
+     * The total count of the chunks.
+     */
+    totalChunks: number;
+    /**
+     * The size of the chunk in bytes.
+     */
+    chunkSize: number;
+    /**
+     * The number of the uploaded chunk.
+     */
+    chunkNumber: number;
+    /**
+     * The ID of the chunk.
+     */
+    chunkId: string;
```

Comment:
Please separate the members with comments with a empty new-line same as you do below for the `UploadOptions`.